client/src/main/scala/org/apache/celeborn/client/commit/CommitHandler.scala

```diff
@@ -195,11 +348,9 @@ abstract class CommitHandler(
 
     val commitFileStartTime = System.nanoTime()
     val workerPartitionLocations = allocatedWorkers.asScala.filter(!_._2.isEmpty)
-    val parallelism = Math.min(workerPartitionLocations.size, conf.clientRpcMaxParallelism)
-    ThreadUtils.parmap(
-      workerPartitionLocations,
-      "CommitFiles",
-      parallelism) { case (worker, partitionLocationInfo) =>
+
+    val params = new util.ArrayList[CommitFilesParam](workerPartitionLocations.size)
+    workerPartitionLocations.foreach({ case (worker, partitionLocationInfo) =>
```

Comment:
eliminate the unnecessary `()`

```
seq.foreach { case x =>
  ...
}
```

there are a few places have same issues

Reply:
done